src/overview/components/Overview.jsx

```diff
@@ -18,7 +19,7 @@ class Overview extends React.Component {
                 ref='inputQuery'
             >
             </input>
-            <ResultList searchResult={this.props.searchResult} />
+            {this.props.loading==0 ? <ResultList searchResult={this.props.searchResult} />: <Loader />  }
```

Comment:
I would remove `==0` (and swap the two values). Non-zero is truthy, and conceptually the variable is pretty much a boolean, that it keeps count is an implementation detail.